tools/nightly-e2e-tests/monitors-gen/terraform.tfstate

```diff
@@ -0,0 +1,225 @@
+{
```

Comment:
💡 I'm not yet sure what to do with this file. Ideally, we should put it on [TF remote backend](https://www.terraform.io/docs/language/settings/backends/remote.html) so that the state is always managed from single machine. I will come with the best answer before merging this PR.

Reply:
Got my opinion on that - proposed simplified Terraform workflow in #521.